nilearn/surface/tests/test_surface.py

```diff
@@ -595,23 +595,6 @@ def test_vol_to_surf(kind, n_scans, use_mask):
         surface.vol_to_surf(img, mesh, interpolation="bad")
 
 
-def test_vol_to_surf_file_freesurfer(tmp_path):
-    # Applies to freesurfer files (.orig, .pial, .white, .sphere, or .inflated)
-    template = datasets.load_mni152_template(resolution=2)
-    mesh = generate_surf()
-    for suff in ['.pial', '.inflated', '.white', '.orig', '.sphere']:
-        fd, filename_fs_mesh = tempfile.mkstemp(suffix=suff,
-                                                dir=str(tmp_path))
-        os.close(fd)
-        nb.freesurfer.write_geometry(filename_fs_mesh, mesh[0], mesh[1])
-        surf_file = filename_fs_mesh
-        mesh = surface.load_surf_mesh(surf_file)
-        mapped = surface.vol_to_surf(template, mesh)
-        normalized = (mapped - min(mapped)) / (max(mapped) - min(mapped))
-        assert np.var(normalized) < 0.02
-        os.remove(filename_fs_mesh)
-
```

Comment:
Sorry, I think I misunderstood what you were doing in the discussion.
So, as the mesh is generated locally you only rely on the template, but this is shipped with Nilearn, hence we should not worry about that, right ? Maybe for the template we could simply use
`from nilearn.datasets import MNI152_FILE_PATH` (see e.g. https://nilearn.github.io/auto_examples/plot_nilearn_101.html)
I would only run it on one surface only, since the test may be a bit expensive. Do we agree that the test would fail on the current codebase ?
And I would add count (refs to the issue) to explain the purpose of the test.

Reply:
Yes this test fails and this was expected, it was just an example of what it should look like. The mesh generated locally is useless because we actually need the freesurfer specific files which we cannot generate and we do not ship with nilearn. 

The issue relates to how the coordinates are read for specific freesurfer files such as .pial and we do not ship those with Nilearn. In other words, the only way I see to test this is if we have a . pial file that we can load with `load_surf_mesh`

Reply:
Indeed we can use the template you mention but still we need a .pial file (or .white, etc)

Reply:
But can't we geenrate the .pial file with nibabel ?

Reply:
But yes, I understand that we can't generate it with the cras thing as freesrfer does. Forget about my latest comment.